content/_data/authors/tracymiranda.adoc

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
You could also add avatar to https://github.com/jenkins-infra/jenkins.io/tree/master/content/images/avatars

Reply:
Good to know - I'll add that in.